web-marketplace/src/lib/rdf/rdf.unknown-fields.ts

```diff
@@ -92,7 +92,8 @@ export function getBatchUnknownFields<
 }
 
 export function getFieldLabel(fieldName: string) {
-  return fieldName.split(':')[1].replace(/([A-Z])/g, ' $1');
+  // Remove the field prefix and add space before capital letters.
+  return fieldName.replace(/[^:]*\:/, '').replace(/([A-Z])/g, ' $1');
```

Comment:
This fixes another bug we discovered where the page crashes if the field key doesn't have a colon prefix. This is a simple change to safely remove the first `prefix:` if one exists.

Compare: https://dev.app.regen.network/credit-batches/MBS01-001-20240601-20340531-002

And: https://deploy-preview-2496--regen-marketplace.netlify.app/credit-batches/MBS01-001-20240601-20340531-002